addons/portal/controllers/mail.py

```diff
@@ -117,11 +22,19 @@ def _portal_post_has_content(self, thread_model, thread_id, message, attachment_
 
     @http.route('/mail/avatar/mail.message/<int:res_id>/author_avatar/<int:width>x<int:height>', type='http', auth='public')
     def portal_avatar(self, res_id=None, height=50, width=50, access_token=None, _hash=None, pid=None):
-        """ Get the avatar image in the chatter of the portal """
+        """Get the avatar image in the chatter of the portal"""
         if access_token or (_hash and pid):
-            message = request.env['mail.message'].browse(int(res_id)).exists().filtered(
-                lambda msg: _check_special_access(msg.model, msg.res_id, access_token, _hash, pid and int(pid))
-            ).sudo()
+            message = (
+                request.env["mail.message"]
+                .browse(int(res_id))
+                .exists()
+                .filtered(
```

Comment:
Broken formatting, could you try to write more than one word / line ?

Reply:
I'm using the black with -l 100. What is the suggestion for formatter?